vpr/src/place/net_cost_handler.cpp

```diff
@@ -448,7 +453,88 @@ static double wirelength_crossing_count(size_t fanout);
 static void set_bb_delta_cost(const int num_affected_nets, double& bb_delta_c);
 
 /******************************* End of Function definitions ************************************/
+// Initialize the ts vectors
+void BB2D3DControlFlow::init(size_t num_nets, bool cube_bb_in) {
+    const int num_layers = g_vpr_ctx.device().grid.get_num_layers();
+
+    cube_bb = cube_bb_in;
+    if (cube_bb) {
+        ts_info.ts_bb_edge_new.resize(num_nets, t_bb());
+        ts_info.ts_bb_coord_new.resize(num_nets, t_bb());
+    } else {
+        VTR_ASSERT_SAFE(!cube_bb);
```

Comment:
I don't have a strong opinion, but usually I don't write asserts that check the else is the opposite of the if condition.